tools/coreml/pip_package/setup.py

```diff
@@ -40,7 +40,7 @@ def readme():
         return f.read()
 
 setup(name='mxnet-to-coreml',
-      version='0.1.0',
+      version='0.1.1',
```

Comment:
https://pypi.org/project/mxnet-to-coreml/ seems to indicate we are already at 0.1.2.
Update this to 0.1.3 instead?

Reply:
Updated